src/dbcp/transform/offshore_wind.py

```diff
@@ -96,56 +173,6 @@ def _add_geocoded_locations(transformed_locs: pd.DataFrame) -> None:
     return
 
 
-def _validate_raw_data(raw_dfs: dict[str, pd.DataFrame]) -> None:
-    proj = raw_dfs["offshore_projects"].copy()
-    locs = raw_dfs["offshore_locations"].copy()
-    for df in (proj, locs):
-        assert (
-            not df.eq("").any().any()
-        ), f"Empty strings in df with index {df.index.name}"
-        assert df.index.is_unique, f"Index {df.index.name} is not unique."
-
-    assert (
-        proj["Size (megawatts)"].lt(100).sum() == 1
-    ), "Found unusually small project. Block Island is the only known exception."
-    assert proj["Size (megawatts)"].max() < 10_000, "Found unusually large project."
-
-    assert (
-        proj["Recipient State"].str.contains("TBD").sum() == 1
-    ), "More missing recipient states than expected."
-
-    assert (
-        proj["Online date"].lt(2022).sum() == 1
-    ), "Found project with erroneously early online_date. Block Island is the only known exception."
-    assert (
-        proj["Online date"].max() < 2040
-    ), "Found project with erroneously late online_date."
-
-    # check referential symmetry (should be handled by Airtable, but check anyway)
-    locations_to_landings = pd.MultiIndex.from_frame(
-        _id_association_table_from_csv_array(locs["Cable project IDs"])
-    )
-    landing_to_locations = pd.MultiIndex.from_frame(
-        _id_association_table_from_csv_array(
-            proj["Cable Location IDs"], name="location_id"
-        )[
-            ["location_id", "project_id"]
-        ]  # reorder columns
-    )
-    assert locations_to_landings.symmetric_difference(landing_to_locations).empty
-
-    locations_to_ports = pd.MultiIndex.from_frame(
-        _id_association_table_from_csv_array(locs["assembly/manufac project IDs"])
-    )
-    ports_to_locations = pd.MultiIndex.from_frame(
-        _id_association_table_from_csv_array(
-            proj["Port Location IDs"], name="location_id"
-        )[["location_id", "project_id"]]
-    )
-    assert locations_to_ports.symmetric_difference(ports_to_locations).empty
-    return
-
-
 def _add_actionable_and_nearly_certain_classification(projects: pd.DataFrame) -> None:
```

Comment:
This code was unchanged but I feel like this should be based on `overall_project_status` rather than `construction_status`. I'll check with Yujia

Reply:
K she said to keep it